spec/rubocop/cop/rspec/predicate_matcher_spec.rb

```diff
@@ -270,6 +272,16 @@
         RUBY
       end
 
+      context 'when custom matchers are allowed' do
+        let(:allowed_explicit_matchers) { ['be_a_custom_matcher'] }
```

Comment:
Maybe give a real example explaining the reasoning behind this, e.g. `have_http_status`?

Reply:
@pirj `have_http_status` is one of build in matchers that potentially could be allowed with this feature. I can change `be_a_custom_matcher` to it if you prefer, that said the usage is much broader, like there are many rspec libraries that implement matchers, but also each project may have it's own custom matchers. Let me know your thoughts.

Reply:
Sure, I understand your point that `have_http_status` may be allowed by default in the future.
However, chances are it will be allowed using the default value for the `AllowedExplicitMatchers` configuration option that you've introduced.
With this in mind, the spec will not only make sure that the option works, but at the same time will also indicate the reasoning behind this option, since it's a widely known matcher.